packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -128,6 +130,15 @@ const Route = ({
     });
   });
 
+  const eventTypePrefix =
```

Comment:
```suggestion
 // /team/{TEAM_SLUG}/{EVENT_SLUG} -> /team/{TEAM_SLUG}
  const eventTypePrefix =
```

A comment would be helpful here.